lib/helpers.coffee

```diff
@@ -1,8 +1,36 @@
 SelectorLinter = require 'atom-selector-linter'
+CSON = require 'season'
+fs = require 'fs-plus'
 
 exports.getSelectorDeprecations = ->
   linter = new SelectorLinter(maxPerPackage: 50)
   linter.checkPackage(pkg) for pkg in atom.packages.getLoadedPackages()
+  
+  userKeymapPath = atom.keymaps.getUserKeymapPath()
+  
+  if fs.isFileSync(userKeymapPath)
+    try
+      userKeymapCson = CSON.readFileSync(userKeymapPath)
```

Comment:
I think you can just call this `userKeymap`, `CSON.readFileSync` supports both `.json` and `.cson` files and `~/.atom/keymap.cson` and `~/.atom/keymap.json` are both supported.